pkgs/tools/package-management/poetry/unwrapped.nix

```diff
@@ -84,12 +85,12 @@ buildPythonPackage rec {
     shellingham
     tomlkit
     trove-classifiers
+    tomli
+    tomli-w
```

Comment:
I don't think `tomli-w` is used in `poetry`, only in `poetry-core`. Also, `tomli` should only be required if the python version is older than 3.11 because in `poetry` uses `tomllib` in 3.11.

https://github.com/python-poetry/poetry/blob/3fbd794ab8ab09bf99f90116c463996031b5544d/pyproject.toml#L59